src/Symfony/Installer/NewCommand.php

```diff
@@ -44,6 +50,7 @@ class NewCommand extends Command
     private $version;
     private $compressedFilePath;
     private $requirementsErrors = array();
+    private $versionsList = null;
```

Comment:
I don't think you need to make it stateful here. It is needed only once in each case anywya